packages/suite/src/components/suite/AccountTypeBadge.tsx

```diff
@@ -0,0 +1,23 @@
+import { AccountType, UppercaseAccountType } from '@suite-common/wallet-types';
+import { toUppercaseType } from '@suite-common/suite-utils';
+import { Badge, BadgeSize } from '@trezor/components';
+import { Translation } from './Translation';
+
+type AccountTypeBadgeProps = {
+    accountType?: AccountType;
+    size?: BadgeSize;
```

Comment:
nit:
Once you use BadgeSize and once BadgeProps[size]

If you use BadgeProps, badgesize does not need to be exported